doc/Development_Documentation/21_Deployment/03_Configuration_Environments.md

```diff
@@ -52,45 +52,63 @@ The following options are available:
 - `disabled` 
   - do not allow to edit/write configs at all
 
-Available environment variables are: 
-```bash
-PIMCORE_WRITE_TARGET_IMAGE_THUMBNAILS=settings-store
-PIMCORE_WRITE_TARGET_CUSTOM_REPORTS=settings-store
-PIMCORE_WRITE_TARGET_VIDEO_THUMBNAILS=settings-store
-PIMCORE_WRITE_TARGET_DOCUMENT_TYPES=settings-store
-PIMCORE_WRITE_TARGET_WEB_TO_PRINT=settings-store
-PIMCORE_WRITE_TARGET_PREDEFINED_PROPERTIES=settings-store
-PIMCORE_WRITE_TARGET_PREDEFINED_ASSET_METADATA=settings-store
-PIMCORE_WRITE_TARGET_STATICROUTES=settings-store
-PIMCORE_WRITE_TARGET_PERSPECTIVES=settings-store
-PIMCORE_WRITE_TARGET_CUSTOM_VIEWS=settings-store
-```
 #### Storage directory for symfony Config files
 
-The default storage directory for Symfony Config files is `/var/config/...`. However you can change
-the directory to any other directory using the corresponding environment variable:
-
-```bash
-PIMCORE_CONFIG_STORAGE_DIR_IMAGE_THUMBNAILS=/var/www/html/var/config/image-thumbnails
-PIMCORE_CONFIG_STORAGE_DIR_CUSTOM_REPORTS=/var/www/html/var/config/custom-reports
-PIMCORE_CONFIG_STORAGE_DIR_VIDEO_THUMBNAILS=/var/www/html/var/config/video-thumbnails
-PIMCORE_CONFIG_STORAGE_DIR_DOCUMENT_TYPES=/var/www/html/var/config/document-types
-PIMCORE_CONFIG_STORAGE_DIR_WEB_TO_PRINT=/var/www/html/var/config/web-to-print
-PIMCORE_CONFIG_STORAGE_DIR_PREDEFINED_PROPERTIES=/var/www/html/var/config/predefined-properties
-PIMCORE_CONFIG_STORAGE_DIR_PREDEFINED_ASSET_METADATA=/var/www/html/var/config/predefined-asset-metadata
-PIMCORE_CONFIG_STORAGE_DIR_STATICROUTES=/var/www/html/var/config/staticroutes
-PIMCORE_CONFIG_STORAGE_DIR_PERSPECTIVES=/var/www/html/var/config/perspectives
-PIMCORE_CONFIG_STORAGE_DIR_CUSTOM_VIEWS=/var/www/html/var/config/custom-views
+The default storage directory for Symfony Config files is `/var/config/...`.
+
+Available options for write targets and directory for Symfony Config files are: 
+```yaml
+pimcore:
+    storage:
+        image_thumbnails:
+            target: 'settings-store'
+            directory: '/var/www/html/var/config/image-thumbnails'
```

Comment:
Maybe we should be more specific here, because the `directory` node doesn't make sense when using `settings-store`. 
We should only use examples here that really make sense and work properly. 

Maybe we should even include examples where the values are set by environment variables, this makes migration easier.